playbooks/openstack-ses_aio_hostconfig.yml

```diff
@@ -19,12 +19,13 @@
     - name: Ensure all the vars are loaded
       include_vars: "{{ playbook_dir }}/../vars/deploy-on-openstack.yml"
 
-    # TODO(evrardjp): Test if we can include this step in the creation of the node.
-    # In the meantime, we rely on raw ansible module to run task, which causes
-    # the absence of idempotency, and therefore complaining of ansible-lint (301)
     - name: Configure host repositories
-      raw: |
-        zypper ar -f -G {{ deploy_on_openstack_repos_to_configure[item] }} {{ item }}
+      zypper_repository:
```

Comment:
This has a bug and is not idempotent https://github.com/ansible/ansible/issues/23257 , check if it doesn't affect us